packages/inter-protocol/test/psm/test-psm.js

```diff
@@ -305,6 +340,108 @@ test('limit', async t => {
   // t.throwsAsync(() => await E(seat1).getOfferResult());
 });
 
+/** @type {[kind: 'want' | 'give', give: number, want: number, ok: boolean, wants?: number][]} */
+const trades = [
+  ['give', 200, 190, false],
+  ['want', 101, 100, true, 1],
+  ['give', 50, 50, false],
+  ['give', 51, 50, true, 1],
```

Comment:
I get what this is doing now, but only because of side conversations about ava macros. It would be nice if it said that the first line was intended to fail because the PSM doesn't have any anchor until after a `want` completes. Does the third fail because it doesn't include fees?